descriptors/ModuleDescriptor-template.json

```diff
@@ -388,6 +388,26 @@
             "consortia.sharing-roles.item.delete"
           ],
           "modulePermissions": []
+        },
+        {
+          "methods": [
+            "POST"
+          ],
+          "pathPattern": "/consortia/{consortiumId}/sharing/roles/capability-sets",
+          "permissionsRequired": [
+            "consortia.sharing-roles-capability-sets.item.post"
+          ],
+          "modulePermissions": []
+        },
+        {
+          "methods": [
+            "DELETE"
+          ],
+          "pathPattern": "/consortia/{consortiumId}/sharing/roles/capability-sets",
```

Comment:
I see api generated like this, so we don't have 'sharing' in api path, also {{roleId}} should be added here?
![image](https://github.com/user-attachments/assets/c168fa00-b2b6-4d66-8507-212a442efa55)